avocado/utils/ssh.py

```diff
@@ -214,3 +214,37 @@ def quit(self):
         :rtype: bool
         """
         return self._master_command('exit')
+
+    def scp_connection(self):
+        try:
+            SCP_CLIENT_BINARY = path_utils.find_command('scp')
+        except path_utils.CmdNotFoundError:
+            SCP_CLIENT_BINARY = None
+        cmd = self._dash_o_opts_to_str(self.DEFAULT_OPTIONS)
+        cmd = "%s %s" % (SCP_CLIENT_BINARY, cmd)
+        return cmd
+
+    def scp_cmd(self, source, destination, reverse=False, recursive=False):
+        """
+        copy file to and from remote through scp session
```

Comment:
This docstring will be displayed in our documentation, so could you please upper the first letter and also finish the sentence with a dot?

Reply:
ok